pkg/forwarder/forwarder_health_test.go

```diff
@@ -55,13 +55,14 @@ func TestHasValidAPIKeyErrors(t *testing.T) {
 	defer ts1.Close()
 	defer ts2.Close()
 
-	keysPerDomains := map[string][]string{
+	keysPerAPIEndpoint := map[string][]string{
```

Comment:
Could we add a quick test that test the convertion from `https://app.datadoghq.com` to `https://api.datadoghq.com` ?